interbtc-spec/docs/source/spec/oracle.rst

```diff
@@ -3,137 +3,122 @@
 
-The Exchange Rate Oracle receives a continuous data feed on the exchange rate between two currencies such as BTC and DOT.
+In the implementation, the :ref:`feedValues` function does not directly update the aggregate - this is done in the :ref:`oracle_onInitialize` hook, in order to keep the ``feedValues`` function weight independent of the number of oracles. Furthermore, for oracle offline detection and for updating the aggregate when a value becomes outdated, the ``onInitialize`` hook was necessary anyway. 
 
-The implementation of the oracle **is not part of this specification**. InterBTC assumes the oracle operates correctly and that the received data is reliable. 
+The implementation of the oracle client **is not part of this specification**. InterBTC assumes the oracle operates correctly and that the received data is reliable. 
 
 
 Data Model
 ~~~~~~~~~~
 
-Scalars
--------
-
-ExchangeRate
-............
+Enums
+-----
 
-The base exchange rate MUST be stored in the smallest denomination of the currency pair (e.g., Planck per Satoshi). This exchange rate is used to determine how much collateral is required to issue a specific amount of interBTC.
+OracleKey
+.........
 
-.. note:: If the exchange rate between BTC and DOT is 2308 (i.e. 1 BTC = 2308 DOT) then we can convert to the base rate as follows:
-    ``planck_per_satoshi = dot_per_btc * (10**dot_decimals / 10**btc_decimals)``
-    ``230800 = 2308 * (10**10 / 10**8)``
+Key to indicate a specific value.
 
-The exchange rate MUST be stored in a 128-bit unsigned fixed-point representation.
+.. tabularcolumns:: |l|L|
 
-SatoshiPerBytes
-...............
+=======================================  ========================================================================
+Discriminant                             Description
+=======================================  ========================================================================
+``ExchangeRate(CurrencyId)``             Exchange rate against Bitcoin, in e.g. planck per satoshi.
+``FeeEstimation(BitcoinInclusionTime)``  Estimate of the bitcoin inclusion fee.
+=======================================  ========================================================================
 
-The estimated Satoshis per bytes required to get a Bitcoin transaction included - see the :ref:`btcTxFeesPerByte`.
-
-MaxDelay
-........
-
-The maximum delay in seconds between incoming calls providing exchange rate data. If the Exchange Rate Oracle receives no data for more than this period, the BTC Parachain enters an ``Error`` state with a ``ORACLE_OFFLINE`` error cause.
-
-LastExchangeRateTime
+BitcoinInclusionTime
 ....................
 
-UNIX timestamp indicating when the last exchange rate data was received. 
+Indicates the time period for bitcoin inclusion fee estimates.
```

Comment:
Perhaps the diff isn't rendering correctly, but this comment is wrong for the table below.

Reply:
Seems correct for me